icons/mob/human_races/cyberlimbs/shellguard/robolimbs.dm

```diff
@@ -0,0 +1,141 @@
+var/global/list/all_robolimbs = list()
```

Comment:
This file shouldn't be necessary as the shellguard datums are defined in the above file  code/modules/surgery/organs/robolimbs.dm